easybuild/easyconfigs/p/PyFR/PyFR-1.7.6-intel-2018a-Python-3.6.4-CUDA-9.1.85.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonPackage'
+
+name = 'PyFR'
+version = '1.7.6'
+versionsuffix = '-Python-%(pyver)s-CUDA-9.1.85'
+
+homepage = 'http://http://www.pyfr.org'
+
+description = """PyFR is an open-source Python based framework for solving
+advection-diffusion type problems on streaming architectures using the Flux
+Reconstruction approach of Huynh. The framework is designed to solve a range of
+governing systems on mixed unstructured grids containing various element types.
+It is also designed to target a range of hardware platforms via use of an
+in-built domain specific language derived from the Mako templating engine."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://www.pyfr.org/download']
+sources = [SOURCE_ZIP]
+checksums = ['a21a7127d8e6f0a6d705ed2fd6a8681e7d00e31065fc54cf06a2dd20b049d6dd']
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('PyCUDA', '2018.1', '-Python-3.6.4-CUDA-9.1.85'),
```

Comment:
@SimonPinches Please avoid hardcoding the Python version in multiple places. You can use `versionsuffix` here, and `-Python-%(pyver)s` below:

```python
dependencies = [
    ('Python', '3.6.4'),
    ('PyCUDA', '2018.1', versionsuffix),
    ('h5py', '2.7.1', '-Python-%(pyver)s'),
    ('Mako', '1.0.7', '-Python-%(pyver)s'),
]